lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3738,7 +3738,10 @@ namespace {
         assert(((decl->getNumParams() == argNames.size() + 1) || isAccessor) &&
                (*selfIdx < decl->getNumParams()) && "where's self?");
       } else {
-        assert(decl->getNumParams() == argNames.size() || isAccessor);
+        // Variadic members, such as "void test(...)" will appear to have no
+        // parameters; that's OK.
+        assert(decl->getNumParams() == argNames.size() || decl->isVariadic() ||
```

Comment:
Wdyt about either:

```
decl->getNumParams() + (decl->isVariadic() ? 1 : 0) || ...
```

or 

```
decl->getNumParams() == argNames.size() || decl->isVariadic() && decl->getNumParams() +1 == argNames.size() || ...
```

Reply:
I like the first one but I think I'll break it out into a variable so it's easier to read.